### DISCUSSION TITLE
Suddenly spell check is treating dashes like part of the word?

### DISCUSSION THREAD

**Discussion by DrReddP** [SubtitleEdit/subtitleedit#5273] [General]
Why, and how do I fix it
![image](https://user-images.githubusercontent.com/82276985/130473787-13c0bc08-26f8-4e98-b303-9d71a8891aa3.png)

**Comment by niksedk** [comment]
Could you attach the subtitle?
Probably not a standard dash.

**Comment by DrReddP** [comment]
How do I check the nature of the dashes for myself?
[TaskmasterNZS02E01.txt](https://github.com/SubtitleEdit/subtitleedit/files/7033313/TaskmasterNZS02E01.txt)

**Comment by niksedk** [comment]
You can try to paste your text into one of the online analyzers, like https://www.fontspace.com/unicode/analyzer

You can also try SE's Tools - Fix common errors - Normalize strings.

Also, SE now updated to handle this better: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.1/SubtitleEditBeta.zip
